packages/react-components/src/shared/src/omitProps.ts

```diff
@@ -1,4 +1,4 @@
-export function omitProps<TObject extends Record<string, any>, TKey extends keyof TObject>(obj: TObject, omit: TKey[]) {
+export function omitProps<TObject extends Record<string, any>, TKey extends string>(obj: TObject, omit: TKey[]) {
```

Comment:
Could you instead do: 

```
export function omitProps<TObject extends Record<string, any>>(obj: TObject, omit: string[]) {

```

Reply:
i can't since i'm using TKey to determine the return value